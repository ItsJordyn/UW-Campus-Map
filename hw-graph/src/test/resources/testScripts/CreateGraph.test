# CreateGraph Testing file
# Tests that a graph can be created

# Create a graph
CreateGraph graph1