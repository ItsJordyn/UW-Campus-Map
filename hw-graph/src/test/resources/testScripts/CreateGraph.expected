# CreateGraph Testing file
# Tests that a graph can be created

# Create a graph
created graph graph1